package.cmake

```diff
@@ -24,10 +24,10 @@ macro(setSofaFilePath name value)
 endmacro()
 ######################
 
-setSofaString(CMAKE_BUILD_TYPE Release)
+message("---------- Setting some options ---------")
 
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-	setSofaString(CMAKE_CXX_FLAGS "-std=c++11")
+    setSofaString(CMAKE_CXX_FLAGS "-std=c++11")
```

Comment:
I didn't know we were compiling in c++11 when building on clang. Why is that?

Reply:
Because Clang version on our CI needs it :)